backend/src/graphql/resolver/AdminResolver.test.ts

```diff
@@ -261,6 +263,216 @@ describe('AdminResolver', () => {
+          await userFactory(testEnv, stephenHawking)
+          await userFactory(testEnv, garrickOllivander)
+        })
+
+        afterAll(async () => {
+          await cleanDB()
+          resetToken()
+        })
+
+        describe('find', () => {
+          const allUsers = {
+            bibi: expect.objectContaining({
+              email: 'bibi@bloxberg.de',
+            }),
+            garrick: expect.objectContaining({
+              email: 'garrick@ollivander.com',
+            }),
+            peter: expect.objectContaining({
+              email: 'peter@lustig.de',
+            }),
+            stephen: expect.objectContaining({
+              email: 'stephen@hawking.uk',
+            }),
+          }
+
+          it('all users by "filters === null"', async () => {
+            await expect(
+              query({
+                query: searchUsers,
+                variables: {
+                  ...variablesWithoutTextAndFilters,
+                },
+              }),
+            ).resolves.toEqual(
+              expect.objectContaining({
+                data: {
+                  searchUsers: {
+                    userCount: 4,
+                    userList: expect.arrayContaining(convertObjValuesToArray(allUsers)),
+                  },
+                },
+              }),
+            )
+          })
+
+          it('all users by "filterByActivated === null && filterByDeleted === null"', async () => {
+            await expect(
+              query({
+                query: searchUsers,
+                variables: {
+                  ...variablesWithoutTextAndFilters,
+                  filters: {
+                    filterByActivated: null,
+                    filterByDeleted: null,
+                  },
+                },
+              }),
+            ).resolves.toEqual(
+              expect.objectContaining({
+                data: {
+                  searchUsers: {
+                    userCount: 4,
+                    userList: expect.arrayContaining(convertObjValuesToArray(allUsers)),
+                  },
+                },
+              }),
+            )
+          })
+
+          it('users with unchecked email', async () => {
+            await expect(
+              query({
+                query: searchUsers,
+                variables: {
+                  ...variablesWithoutTextAndFilters,
+                  filters: {
+                    filterByActivated: false,
+                    filterByDeleted: null,
+                  },
+                },
+              }),
+            ).resolves.toEqual(
+              expect.objectContaining({
+                data: {
+                  searchUsers: {
+                    userCount: 1,
+                    userList: expect.arrayContaining([allUsers.garrick]),
+                  },
+                },
+              }),
+            )
+          })
+
+          it('users with deleted account', async () => {
```

Comment:
New describe block.
`filter by deleted users`

```suggestion
          it('finds only users with deleted account', async () => {
```

Reply:
Cool! 😍
Done.